src/katgpucbf/__init__.py

```diff
@@ -39,10 +39,13 @@
 # is better to minimise the number of packets/second to process.
 DEFAULT_PACKET_PAYLOAD_BYTES: Final = 8192
 DEFAULT_TTL: Final = 4  #: Default TTL for spead multicast transmission
-DEFAULT_KATCP_HOST = ""  # All interfaces
-DEFAULT_KATCP_PORT = 7147
-DIG_HEAP_SAMPLES = 4096
+DEFAULT_KATCP_HOST: Final = ""  # All interfaces
+DEFAULT_KATCP_PORT: Final = 7147
+DIG_HEAP_SAMPLES: Final = 4096
 DIG_SAMPLE_BITS: Final = 10
+# Minimum update period (in seconds) for katcp sensors where the underlying
+# value may update extremely rapidly.
+MIN_SENSOR_UPDATE_PERIOD: Final = 1.0
```

Comment:
If I want to be *really* pedantic (and occasionally I do), I'd recommend that the unit be included in the name. I see you have documented it in the comments, so I'm not going to make an issue of it.

I see that this file (and subsequently these constants) don't seem to make their way into whatever Sphinx outputs, at least I can't find them. That may be something worth thinking about.

Reply:
> If I want to be really pedantic (and occasionally I do), I'd recommend that the unit be included in the name.

I can see merit in that where the unit isn't obvious e.g. not the standard SI unit. If I see a variable holding time/frequency/length/power, I assume it is in seconds/Hertz/metres/watts.

Reply:
> I see that this file (and subsequently these constants) don't seem to make their way into whatever Sphinx outputs, at least I can't find them. That may be something worth thinking about.

Fixed now.